app/assets/config/manifest.js

```diff
@@ -1,4 +1,5 @@
 //= link_tree ../images
+//= link_tree ../videos
```

Comment:
The directory /videos was added to the manifest.js tree.

This is so that on precompilation, the main homebanner video will be included in the asset pipeline.